### ISSUE TITLE
Feature consideration: using arbitrary directories as upload destination

### ISSUE THREAD

**Issue by vakabus** [vakabus/gimmedat#12]
I am thinking about integration with other services running on the same host server...

Imagine, we have a web gallery for photos running on the server. It uses a simple directory as its datastore and anything you copy there will be available in the web interface when you load the webpage. (e.g. [PiGallery](https://bpatrik.github.io/pigallery2/)). There is also gimmedat running on that server.

Is there a reasonably general and meaningful way, how to make gimmedat and the gallery work together? I want to be able to tell my friends: "if you upload the files, they will all be visible in the gallery on this link". And ideally, I would like to be able to set it up on my phone.

In other words, the goal would be:
- somehow make sure that the uploaded files are accessible from specified data directories elsewhere (not necessarily in cwd)
- configure the destination of files from the web UI

Questions:
- Isn't this out of scope?
- How to do it?

**Comment by vakabus** [comment]
> Isn't this out of scope?

I have not decided yet. I think that we should take care of this, however at the same time, letting the app store files anywhere on the filesystem sounds dangerous and like something it shouldn't be able to do.

> How to do it?

Two ideas:

- Symlinks are a working solution which we do not have to implement. The only thing missing would be the web UI to set them up and that sounds a bit messy.

- We could add command line arguments specifying named "destinations" and we could let the user select one "destination" when creating the upload link. That would still prevent gimmedat from placing files just about anywhere, but at the same time, this feature would allow us to configure more storage directories and decide in runtime which one to use.
